### PR TITLE
Set up new pages

### PR DESCRIPTION
**Pull request by Camel-app**
first final version of dashboard :-)

### CODE REVIEW

File: components/apply/Apply.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    </div>
+    </Container>
+    </div>
+    );	
+}
+
+export default ApplyAccount;
+
+
+
```
Comment (Aodarium) [inline components/apply/Apply.tsx:126]: I would space the different parts a bit more - it is too compact in comparison to other pages

File: components/experiments/Experiments.tsx [hunk 2]
```diff
@@ -56,6 +79,36 @@ const Experiments = ({ data }) => {
...
+				<List.Item>
+					Status: set the status of your experiment (it is only possible to participate in active experiments)
+				</List.Item>
+			</List>
+			</div>
+			For detailed information  how to set up an experiment have a look in our <a key='Documentation Set Up' href='https://camtools-documentation.readthedocs.io/en/master/Set%20up%20study/#general-procedure' target='_blank'>
+                                <span className={classes.internalLinks}>
+                                Documentation
+                                </span>
+                                </a>
```
Comment (Aodarium) [inline components/experiments/Experiments.tsx:111]: I would leave this to the documentation part - it does not carry relevant information with the current design

File: components/footer/Footer.tsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ export default function FooterCentered({ links }: FooterCenteredProps) {
       key={link.label}
       href={link.link}
       sx={{ lineHeight: 1 }}
-      onClick={(event) => event.preventDefault()}
+      //onClick={(event) => event.preventDefault()}
       size="sm"
     >
       {link.label}
```
Comment (Aodarium) [inline components/footer/Footer.tsx:45]: Is it possible to center the footer?

File: components/information/Information.tsx [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+const Information = () => {
+    const { classes } = useStyles();
+    return (
+        <div>
+            <Container>
+            <div className={classes.inner}>
+                    <div className={classes.contentBelowImage}>
+                        <Title className={classes.title}>
+                        Thank you for checking out our <br></br> <span className={classes.highlight}>Cognitive Affective Map tools</span>!
+                        </Title>
```
Comment (Aodarium) [inline components/information/Information.tsx:114]: The "C" of Cognitive is not aligned with the T

File: components/information/Information.tsx [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+                    <div className={classes.inner}>
+                    <div>
+                            <Image src={image.src} className={classes.image} />
+                            </div>
+                        <div className={classes.contentNextImage}>
+                            <List
+                                    mt={30}
+                                    spacing="sm"
+                                    size="sm"
```
Comment (Aodarium) [inline components/information/Information.tsx:129]: I would increase the font, it s a bit small to read

File: components/information/Information.tsx [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+                                    </List.Item>
+                                    <List.Item>
+                                        Thanks to our ready-made example studies, constructing studies couldn't be easier (see <a key='example studies' href=' https://camtools-documentation.readthedocs.io/en/master/Set%20up%20study/#example-studies' target='_blank'>
+                                <span className={classes.internalLinks}>
+                                example studies
+                                </span>
+                                </a>).
+                                    </List.Item>
+                                    <List.Item>
+                                        Using the visual interface of the CAM-App data can be easly summarized and analyzed (see <a key='CAM-App documentation' href='https://camtools-documentation.readthedocs.io/en/master/CAM-App/' target='_blank'>
```
Comment (Aodarium) [inline components/information/Information.tsx:185]: In general, links should be in black, not blue - the underline or bold aspect can stay

File: pages/impressum/index.tsx [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import type { NextPage } from "next";
+
+import HeaderSimple from '../../components/header/Header';
+import Impressum from "../../components/impressum/Impressum";
+import getNavbar from '../../controllers/headerFetcher';
```
Comment (Aodarium) [inline pages/impressum/index.tsx:5]: Not needed here

File: pages/impressum/index.tsx [hunk 8]
```diff
@@ -0,0 +1,16 @@
+import type { NextPage } from "next";
+
+import HeaderSimple from '../../components/header/Header';
```
Comment (Aodarium) [inline pages/impressum/index.tsx:3]: Not needed here

File: pages/information/index.tsx [hunk 9]
```diff
@@ -0,0 +1,17 @@
+import type { NextPage } from "next";
+
+import HeaderSimple from '../../components/header/Header';
```
Comment (Aodarium) [inline pages/information/index.tsx:3]: Not needed here

File: pages/information/index.tsx [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+import Information from "../../components/information/Information";
+import getNavbar from '../../controllers/headerFetcher';
```
Comment (Aodarium) [inline pages/information/index.tsx:5]: Not needed here

File: components/information/Information.tsx (hunk 5 above)
Comment (Aodarium) [reply components/information/Information.tsx:129]: Not only for this line but also ones around

File: components/apply/Apply.tsx (hunk 1 above)
Comment (Camel-app) [reply components/apply/Apply.tsx:125]: solved

File: components/experiments/Experiments.tsx (hunk 2 above)
Comment (Camel-app) [reply components/experiments/Experiments.tsx]: solved

File: components/footer/Footer.tsx (hunk 3 above)
Comment (Camel-app) [reply components/footer/Footer.tsx:46]: would not do it

File: components/information/Information.tsx (hunk 4 above)
Comment (Camel-app) [reply components/information/Information.tsx:120]: solved

File: components/information/Information.tsx (hunk 5 above)
Comment (Camel-app) [reply components/information/Information.tsx:135]: solved

File: components/information/Information.tsx (hunk 6 above)
Comment (Camel-app) [reply components/information/Information.tsx:193]: solved

File: pages/impressum/index.tsx (hunk 7 above)
Comment (Camel-app) [reply pages/impressum/index.tsx]: solved

File: pages/impressum/index.tsx (hunk 8 above)
Comment (Camel-app) [reply pages/impressum/index.tsx]: solved

File: pages/information/index.tsx (hunk 9 above)
Comment (Camel-app) [reply pages/information/index.tsx]: solved

File: pages/information/index.tsx (hunk 10 above)
Comment (Camel-app) [reply pages/information/index.tsx]: solved

**Pull request closed without merging**